src/qibojit/backends/cpu.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+from numba import njit
```

Comment:
Btw, this is weird because this backend is not touched by this branch. But this line is new, and it seems unused.

Let me add pyclean to pre-commit.

Reply:
Ok, pycln was already there, but not detecting these. Pretty weird...